suite-native/module-settings/src/screens/SettingsViewOnly.tsx

```diff
@@ -1,33 +1,117 @@
+            <Translation
+                id="moduleSettings.viewOnly.subtitle"
+                values={{
+                    about: chunk =>
+                        chunk && (
+                            <Link
+                                label={chunk}
+                                onPress={onPressAbout}
+                                isUnderlined
+                                textColor="textSubdued"
+                                textPressedColor="textSubdued"
+                            />
+                        ),
+                }}
+            />
+        </Text>
+    </Box>
+);
+
+const svgWrapperStyle = prepareNativeStyle(utils => ({
+    width: 180,
+    height: 156,
+    paddingBottom: utils.spacings.large,
+}));
+
+const emptyContentStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    paddingHorizontal: utils.spacings.large,
+}));
+
+const EmptyContent = ({ onPressAbout }: AboutProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Box style={applyStyle(emptyContentStyle)}>
+            <Box style={applyStyle(svgWrapperStyle)}>
+                <SecuredCoinsSvg />
+            </Box>
+            <Text variant="titleSmall" textAlign="center">
+                <Translation id="moduleSettings.viewOnly.emptyTitle" />
+            </Text>
+            <About onPressAbout={onPressAbout} />
+        </Box>
+    );
+};
+
+const DevicesContent = ({ onPressAbout }: AboutProps) => {
     const dispatch = useDispatch();
     const { translate } = useTranslate();
-
     const devices = useSelector(selectPhysicalDevices);
+    const { showAlert, hideAlert } = useAlert();
+    const { showToast } = useToast();
 
-    const handleViewOnlyChange = (device: TrezorDevice) => {
+    const toggleViewOnly = (device: TrezorDevice) => {
+        const toast =
+            device.remember ?? false
+                ? 'moduleSettings.viewOnly.toast.disabled'
+                : 'moduleSettings.viewOnly.toast.enabled';
+        showToast({
+            variant: 'default',
+            message: <Translation id={toast} />,
+            icon: 'check',
+        });
         dispatch(toggleRememberDevice({ device }));
     };
 
+    const handleDisableViewOnly = (device: TrezorDevice) => {
+        showAlert({
+            title: translate('moduleSettings.viewOnly.disableDialog.title', { name: device.name }),
+            description: <Translation id="moduleSettings.viewOnly.disableDialog.subtitle" />,
+            primaryButtonTitle: (
+                <Translation id="moduleSettings.viewOnly.disableDialog.buttons.primary" />
+            ),
+            onPressPrimaryButton: () => toggleViewOnly(device),
+            primaryButtonVariant: 'redBold',
+            secondaryButtonTitle: (
+                <Translation id="moduleSettings.viewOnly.disableDialog.buttons.secondary" />
+            ),
+            onPressSecondaryButton: hideAlert,
```

Comment:
```suggestion
            
```

This is redundant. both onPress callbacks of this function close the sheet by default.

Reply:
59fad99c60